CalculatorWithTests/Calculator.cs

```diff
@@ -1,8 +1,47 @@
-﻿
+﻿using System;
+using CalculatorWithTests;
+using System.Linq;
+
+
+
 namespace CalculatorWithTests
 {
-    class Calculator
+    public class Calculator
     {
-        
+        public int Add(int number1, int number2)
+        {
+            throw new NotImplementedException();
+        }
+        public int Subtract(int sub1, int sub2)
+        {
+            throw new NotImplementedException();
+        }
+        public int Multiply(int multiply1, int multiply2)
+        {
+            throw new NotImplementedException();
+        }
+        public int Divide(decimal dividend, decimal divisor)
+        {
+            throw new NotImplementedException();
+        }
+        public int Factorial(long factNumber)
+        {
+            throw new NotImplementedException();
+        }
+
+
+
+
+
+
+
+
```

Comment:
There are a number of blank lines at the bottom of this class that could be removed